lhctl/cmd/tenant.go

```diff
@@ -40,7 +40,24 @@ var searchTenantCmd = &cobra.Command{
 	},
 }
 
+var getTenantCmd = &cobra.Command{
+	Use:   "tenant <id>",
+	Short: "Get a Tenant",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			log.Fatal("You must provide one argument: the tenant Id")
+		}
```

Comment:
You can let cobra handle this for you with 

```diff
var getTenantCmd = &cobra.Command{
	Use:   "tenant <id>",
	Short: "Get a Tenant",
+       Args: cobra. ExactArgs(1),
	Run: func(cmd *cobra.Command, args []string) {
- 		if len(args) != 1 {
- 			log.Fatal("You must provide one argument: the tenant Id")
- 		}
		common.PrintResp(getGlobalClient(cmd).GetTenant(
			requestContext(cmd),
			&model.TenantId{
				Id: args[0],
			},
		))
	},
}
```

Reply:
Thanks, @mijailr ! Will update it before merging.